docOps/docs/libraries/python/api_reference.md

```diff
@@ -0,0 +1,1080 @@
+
+**Returns** the event id as list[int].
+
+### remove_transfer_progress_listener(list[int]): void
+
+Removes the transfer progress listener associated with the given identifier.
+
+| Param | Type                   | Default                | Description  |
+| ----- | ---------------------- | ---------------------- | ------------ |
+| [id]  | <code>list[int]</code> | <code>undefined</code> | The event id |
+
+### on_migration_progress(callback): list[int]
+
+Listen to migration progress events.
+
+| Param      | Type                  | Default                | Description           |
+| ---------- | --------------------- | ---------------------- | --------------------- |
+| [callback] | <code>function</code> | <code>undefined</code> | The callback function |
+
+**Returns** the event id as list[int].
+
+### remove_migration_progress_listener(list[int]): void
+
+Removes the migration progress listener associated with the given identifier.
+
+| Param | Type                   | Default                | Description  |
+| ----- | ---------------------- | ---------------------- | ------------ |
+| [id]  | <code>list[int]</code> | <code>undefined</code> | The event id |
+
+## WalletAddress
+
+A dict with the following key/value pairs.
+
+```python
+wallet_address = {
+    'address': str,
+    'balance': int,
+    'key_index': int,
+    'internal': bool,
+    'outputs': dict[(string, WalletAddressOutput)],
+}
+```
+
+Please refer to [WalletAddressOutput](#walletaddressoutput) for the details of this type.
+
+## WalletAddressOutput
+
+A dict with the following key/value pairs.
+
+```python
+wallet_address_output = {
+    'transaction_id': str,
+    'message_id': str,
+    'index': int,
+    'amount': int,
+    'is_spent': bool,
+    'address': str,
+    'kind': str,
+}
+}
+```
+
+## Address
+
+A dict with the following key/value pairs.
+
+```python
+address = {
+    'address': AddressWrapper,
+    'balance': int,
+    'key_index': int,
+    'internal': bool,
+    'outputs': list[AddressOutput],
+}
+```
+
+Please refer to [AddressWrapper](#addresswrapper) and [AddressOutput](#addressoutput) for the details of this type.
+
+## AddressWrapper
+
+A dict with the following key/value pairs.
+
+```python
+address_wrapper = {
+    'inner': str
+}
+```
+
+## AddressOutput
+
+A dict with the following key/value pairs.
+
+```python
+address_output = {
+    'transaction_id': str,
+    'message_id': str,
+    'index': int,
+    'amount': int,
+    'is_spent': bool,
+    'address': AddressWrapper,
+}
+```
+
+Please refer to [AddressWrapper](#addresswrapper) for the details of this type.
+
+## AccountBalance
+
+A dict with the following key/value pairs.
+
+```python
+account_balance = {
+    'total': int,
+    'available': int,
+    'incoming': int,
+    'outgoing': int,
+}
+```
+
+## ClientOptions
+
+A dict with the following key/value pairs.
+
+```python
+client_options = {
+    'nodes': list[[Node](#node)] (optional),
```

Comment:
<!--StartFragment-->    'primary_node': [Node](#node)] (optional),    'primary_pow_node': [Node](#node)] (optional),<!--EndFragment-->

is missing